chain/network/src/routing.rs

```diff
@@ -22,237 +20,22 @@ use near_store::{
 
 use crate::metrics;
 use crate::{
-    cache::RouteBackCache,
     types::{PeerIdOrHash, Ping, Pong},
     utils::cache_to_hashmap,
 };
 use conqueue::{QueueReceiver, QueueSender};
 #[cfg(feature = "delay_detector")]
 use delay_detector::DelayDetector;
 
+pub use near_network_primitives::routing::*;
```

Comment:
Thinking about this `*`, I think the owners of near-network should revisit the structure and draw a better line between `near-network-primitives` and `near-network` since currently, having two `routing.rs` files it is not immediately clear which one should be extended with the new types (I am talking about hypothetical future need to extend `near-netwok`)

Reply:
@pmnoxx please take a look